spring-context/src/main/java/org/springframework/scheduling/annotation/Scheduled.java

```diff
@@ -101,52 +102,76 @@
 	String zone() default "";
 
 	/**
-	 * Execute the annotated method with a fixed period in milliseconds between the
+	 * Execute the annotated method with a fixed period between the
 	 * end of the last invocation and the start of the next.
-	 * @return the delay in milliseconds
+	 * Using milliseconds by default with fixedDelayTimeUnit().
+	 * @return the delay
 	 */
 	long fixedDelay() default -1;
 
 	/**
-	 * Execute the annotated method with a fixed period in milliseconds between the
+	 * Execute the annotated method with a fixed period between the
 	 * end of the last invocation and the start of the next.
-	 * @return the delay in milliseconds as a String value, e.g. a placeholder
+	 * Using milliseconds by default with fixedDelayTimeUnit().
+	 * @return the delay as a String value, e.g. a placeholder
 	 * or a {@link java.time.Duration#parse java.time.Duration} compliant value
 	 * @since 3.2.2
 	 */
 	String fixedDelayString() default "";
 
 	/**
-	 * Execute the annotated method with a fixed period in milliseconds between
+	 * Specify the {@link TimeUnit} to use for the fixedDelay and the fixedDelayString values.
+	 * @return the {@link TimeUnit}, by default milliseconds will be used.
+	 */
+	TimeUnit fixedDelayTimeUnit() default TimeUnit.MILLISECONDS;
```

Comment:
Please introduce a single `TimeUnit timeUnit() default TimeUnit.MILLISECONDS;` attribute and get rid of the other `TimeUnit` attributes.

Reply:
Done.